spec/steps/execute.fmf

```diff
@@ -45,6 +45,28 @@ example: |
             how: beakerlib
             isolate: true
 
+/avc_check:
+    summary: Check for SELinux messages, default: true
+    description:
+        Optional boolean attribute `avc_check` can be used to
+        disable check for SELinux messages.
+    example: |
+        execute:
+            how: beakerlib
+            avc_check: false
```

Comment:
I really don't know what the `avc_check` should do/mean.  Is it a capability of Beakerlib.

@psss do we wan't this to be default for every beakerlib invocation?

Reply:
Beakerlib has nothing to do with this, it's feature of executor